### PR TITLE
Corrected IaC naming convention notes

### PR DESCRIPTION
**Pull request by eddie-knight**

### CODE REVIEW

File: accelerators/README.md [hunk 1]
```diff
@@ -70,16 +70,18 @@ pending relocation).
...
-- Underscore delimiters
-- First value is an abbreviation of the IaC technology name
-- Second value is an abbreviation of the control standard
-- Third and final value is the service name, matching the directory name
-- Format:
-`<technology-abbreviation>_<control-standard>_<cloud-provider>_<service-name>`
-- Example: `tf_cdmc_gcp_gke`
+- Module repositories must use this three-part name format, 
+- where <NAME> reflects the type of infrastructure the module manages
+- and <PROVIDER> is the main provider where it creates that infrastructure.
```
Comment (abdullahgarcia) [inline accelerators/README.md:82]: CLOUD_PROVIDER

File: accelerators/README.md [hunk 2]
```diff
@@ -70,16 +70,18 @@ pending relocation).
 If the code has not yet been extracted, it should be in a subdirectory.
 Otherwise, it should be linked as a git submodule.
 
+#### Naming Conventions for IaC
+
 Be sure to follow the technology-specific registry guidelines in the event
```
Comment (abdullahgarcia) [inline accelerators/README.md:75]: Maybe:

"Please follow the technology-specific registry guidelines below."

**Review by abdullahgarcia — COMMENTED** [review]
Just left a few observations for you to consider.

File: accelerators/README.md (hunk 2 above)
Comment (eddie-knight) [reply accelerators/README.md:75]: Can you help me understand what is unclear/undesirable about the current phrasing? 

> follow the technology-specific registry guidelines in the event that they conflict with the guidelines noted below

File: accelerators/README.md (hunk 1 above)
Comment (eddie-knight) [reply accelerators/README.md]: applied

File: accelerators/README.md (hunk 2 above)
Comment (abdullahgarcia) [reply accelerators/README.md:75]: Being a non-native English speaker, I found it a bit confusing, specifically the part: "in the event that they conflict with the guidelines noted below". 

The suggestion above is, in my view, more straightforward.

Comment (thinkl33t) [reply accelerators/README.md:75]: I think your suggestion flips the meaning of the sentence, so definitely could be clearer.  How about something like:

"Please follow the naming guidelines below, unless If the technology-specific guidelines conflict.  If there is a conflict, the technology-specific guidelines should be preferred"

**Comment by eddie-knight** [comment]
/easy-cla is updated and should be good to go

**Comment by abdullahgarcia** [comment]
@eddie-knight , I think the EasyCLA is still broken.

**Pull request merged**